include/RAJA/policy/sycl/launch.hpp

```diff
@@ -149,14 +150,14 @@ struct LaunchExecute<RAJA::sycl_launch_t<async, 0>>
 
 
       auto combiner = [](ReduceParams x, ReduceParams y) {
-        RAJA::expt::ParamMultiplexer::combine<EXEC_POL>(x, y);
+        RAJA::expt::ParamMultiplexer::params_combine(EXEC_POL {}, x, y);
```

Comment:
Could you pass in `pol`?

Reply:
I think the combiner is not supposed to capture anything.

Reply:
Right now there isn't any runtime info in the policies so it only really matters that the type is right. It would be nice to pass the same policy object in though.